frontend/src/containers/__tests__/TopicareaSettings.test.tsx

```diff
@@ -184,3 +184,10 @@ test("filters topic areas based on createdBy search input", async () => {
   // Topic area one should appear
   expect(topicarea).toBeInTheDocument();
 });
+
+test("topic area settings should match snapshot", async () => {
+  const wrapper = render(<TopicareaSettings />, {
+    wrapper: MemoryRouter,
+  });
+  expect(wrapper.container).toMatchSnapshot();
+});
```

Comment:
I'm not a fan of snapshot testing in Containers. It makes the snapshot fragile to break if any of the underlying components change, which can be many. I think we should try to keep snapshots only for components and use them sporadically. When possible, we should favor basic assertions with react-testing-library. 

What do you think?

Reply:
Yes, I was surprised when I noted we dont have snapshot tests for containers. I really dont see a good reason why not to have it. If an underlying component changes I think it is fine to update the containers too, anyway update a snapshot test is fairly easy and the alternative of not having them, avoid to snapshot test almost the whole container. Of course ideally the containers should have only other components but in practice that is not always the case. Another area to improve is to create more components out of the containers, though.

Reply:
Let's have a team discussion on this. I don't think we need snapshot testing everywhere but let's discuss it.